src/adapter/etl-adapter-excel/src/Flow/ETL/Adapter/Excel/ExcelExtractor.php

```diff
@@ -52,10 +52,6 @@ public function extract(FlowContext $context) : \Generator
         // Offset must be a positive number
         $offset = $this->offset ?? 1;
 
-        if (!$this->withHeader) {
-            $offset++;
-        }
```

Comment:
that was actually a bug 🤦‍♂️

Reply:
Yes, it was missed, and introduced with the above fallback to `1` as offset, cause underlying library has an iterator iterating not from `0` (as it's normal in PHP) but from `1`...